components/StyledInputTags.js

```diff
@@ -153,6 +153,7 @@ const StyledInputTags = ({ suggestedTags, value, onChange, renderUpdatedTags, de
 
   const handleToggleInput = () => {
     isOpen ? handleClose() : setOpen(true);
+    setTimeout(() => inputRef?.current?.focus(), 50);
```

Comment:
@aeworxet : A minor comment, I think it's best to only execute this line when `isOpen` is evaluating to false. We only really need to focus on this input when we open it. 😄

Reply:
Yes, that would be a better practice indeed.

Should I write it in form of a standard `if...else`
```
const handleToggleInput = () => {
  if (isOpen) {
   handleClose();
  } else {
    setOpen(true);
    setTimeout(() => inputRef?.current?.focus(), 50);
  }
};
```
or in form of a ternary
```
const handleToggleInput = () => {
  isOpen ? handleClose() : (setOpen(true), setTimeout(() => inputRef?.current?.focus(), 50));
};
```
?

I see both forms are used in `./components/StyledInputTags.js`, but usual `if...else` would be easier for future maintainers to understand.

Reply:
> I see both forms are used in ./components/StyledInputTags.js, but usual if...else would be easier for future maintainers to understand.

Then first version (if / else) is to be preferred :wink:

Reply:
Commit amended.